src/js/components/Form/stories/CustomThemed/Custom.js

```diff
@@ -64,12 +64,14 @@ export const Custom = () => (
           onReset={(event) => console.log(event)}
           onSubmit={({ value }) => console.log('Submit', value)}
         >
-          <FormField label="Name" name="name" required>
-            <TextInput name="name" />
+          <FormField htmlFor="name" label="Name" name="name" required>
+            <TextInput id="name" aria-required name="name" />
```

Comment:
Not something we need to do now, but it has always bugged me that we have to specify `name` in both places and that I'm not sure many people understand the `htmlFor`/`id` combo is and id which probably should be more unique than this and that the `name`/`name` combo is something much different and really should be this simple (helps form auto-fill really.) Our examples don't really help reinforce this since they tend to use the same string in all four places

Reply:
@jcfilben 
agree here. I think this is a good point and something we should keep in mind.

Reply:
Yeah agree, don't need to change it in this PR but definitely something for the future